hibernate-cache-local/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+   <artifactId>infinispan-simple-tutorials-hibernate-cache-local</artifactId>
+   <parent>
+      <relativePath>../pom.xml</relativePath>
+      <version>1.0.0-SNAPSHOT</version>
+      <groupId>org.infinispan.tutorial.simple</groupId>
+      <artifactId>infinispan-simple-tutorials</artifactId>
+   </parent>
+   <name>Infinispan Simple Tutorials: Hibernate Cache Local</name>
+
+   <properties>
+      <version.hibernate-core>5.2.7.Final</version.hibernate-core>
+      <version.h2>1.3.176</version.h2>
+      <version.log4j>2.8.1</version.log4j>
+   </properties>
+
+   <build>
+      <plugins>
+         <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <executions>
+               <execution>
+                  <goals>
+                     <goal>exec</goal>
+                  </goals>
+               </execution>
+            </executions>
+            <configuration>
+               <executable>java</executable>
+               <arguments>
+                  <argument>-Djava.net.preferIPv4Stack=true</argument>
+                  <argument>-Djava.util.logging.config.file=src/main/resources/logging.properties</argument>
+                  <argument>-classpath</argument>
```

Comment:
I think that you're missing `-ea` switch.

Reply:
Didn't add it since assertions are no longer needed to verify counts.